### ISSUE TITLE
Python snowboydecoder.py multiple callbacks

### ISSUE THREAD

**Issue by webworxshop** [Kitt-AI/snowboy#351]
According to line 195 in snowboydecoder.py only the 0th callback will ever be called, even if multiple callbacks are registered. Changing the line to:

```python
callback = detected_callback[status-1]
```
Fixes this issue. Happy to submit a PR for this if required.

**Comment by chenguoguo** [comment]
Good catch! Please do. Please update the corresponding files in both Python and Python3. Thanks!
